offsite_payments.gemspec

```diff
@@ -23,7 +23,7 @@ Gem::Specification.new do |s|
   s.require_path = 'lib'
 
   s.add_dependency('activesupport', '>= 3.2.14', '< 5.2')
-  s.add_dependency('i18n', '~> 0.5')
+  s.add_dependency('i18n', '~> 0.6.6')
```

Comment:
I think you forgot to change the pessimistic version operator to the one you used in the title ;) i18n is at 0.9.1 at the time of writing.